src/main/java/uk/gov/ons/ctp/response/casesvc/message/CaseReceiptReceiver.java

```diff
@@ -1,15 +1,81 @@
 package uk.gov.ons.ctp.response.casesvc.message;
 
+import static uk.gov.ons.ctp.response.casesvc.representation.CategoryDTO.CategoryName.OFFLINE_RESPONSE_PROCESSED;
```

Comment:
wht the need of static import for just 3 Enum case ? Seems an overkill and is normally discouraged as clan code practice.

Reply:
it's what Google style mandates. This code has been moved from CaseReceiptReceiverImpl.java and is otherwise unchanged